types/test-types.tsx

```diff
@@ -163,7 +164,9 @@ function TestComponent() {
     }
   };
 
-  const [{ price, loading }, setPricing] = useCheckoutPricing(checkoutPricingInput);
+  useCheckoutPricing(checkoutPricingInput);
```

Comment:
I added this line to assert that `useCheckoutPricing` can be called without a `handleError function`